packages/hardhat-core/test/internal/hardhat-network/stack-traces/test.ts

```diff
@@ -673,9 +679,17 @@ const solidity06Compilers: CompilerOptions[] = [
 ];
 
 const solidity07Compilers: CompilerOptions[] = [
+  // {
+  //   solidityVersion: "0.7.0",
+  //   compilerPath: "soljson-v0.7.0+commit.9e61f92b.js",
+  // },
+  // {
+  //   solidityVersion: "0.7.1",
+  //   compilerPath: "soljson-v0.7.1+commit.f4a555be.js",
+  // },
```

Comment:
I'd also add 0.7.2 and 0.7.3, commented out, to make running regression tests easier.